notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>Nice lab! I added some comments, mostly around wording in the markdown. One thought: I think we should (somewhere in the computer vision deep dive) have an example of deploying an image model to a Vertex Endpoint, that has a signature to serve predictions on a base-64 encoded string of an image. I believe this is the recommended way for low latency, online predictions of image data. </p><p> </p><p>Maybe we could show and include a <code>serving_default_b64</code> when deploying to a Vertex Endpoint instead of the <code>serving_default_filenames</code> that is used for batch predictions? Any thoughts on this?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Thank you for the feedback!</p><p>I added the base64 section with additional serving function.</p><p>Could you take a look again?</p>